docs/approach.rst

```diff
@@ -208,10 +208,18 @@ Here we can see time series for some example components (we don't really care ab
 .. image:: /_static/a11_pca_component_timeseries.png
 
 These components are subjected to component selection, the specifics of which
-vary according to algorithm.
+vary according to algorithm. Speficically, ``tedana`` offers two different approaches that perform this step.
 
-In the simplest approach, ``tedana`` uses Minka’s MLE to estimate the
-dimensionality of the data, which disregards low-variance components (the `mle` option in for `--tedpca`).
+The default approach (the `mdl` option for `--tedpca`) is based on a Moving Average (stationary Gaussian) process
+proposed by `Li et al (2007)`_. A moving average process is the output of a linear system (which in this case is
+a smoothing filter) that has an i.i.d. Gaussian process as the input. If we assume that the linear system is shift
```

Comment:
Could we clarify what i.i.d. is here?

Reply:
Sure, I’ll use the complete name. Thanks for the feedback! 🙌🏻